docs/articles_en/learn_openvino/openvino_samples/get_started_demos.md

```diff
@@ -7,293 +7,330 @@
-
-* List the models available in the downloader.
-
-  .. code-block:: sh
-
-     omz_info_dumper --print_all
-
-* Use ``grep`` to list models that have a specific name pattern (e.g. ``ssd-mobilenet``, ``yolo``). Replace ``<model_name>`` with the name of the model.
-
-  .. code-block:: sh
-
-     omz_info_dumper --print_all | grep <model_name>
-
-* Use Model Downloader to download models. Replace ``<models_dir>`` with the directory to download the model to and ``<model_name>`` with the name of the model.
-
-  .. code-block:: sh
-
-     omz_downloader --name <model_name> --output_dir <models_dir>
-
-This guide used the following model to run the Image Classification Sample:
-
-+------------------+-----------------------------+
-| Model Name       | Code Sample or Demo App     |
-+==================+=============================+
-| ``googlenet-v1`` | Image Classification Sample |
-+------------------+-----------------------------+
-
-.. dropdown:: Click to view how to download the GoogleNet v1 Caffe model
+Once the prerequisites have been installed, perform the following steps:
 
-   To download the GoogleNet v1 Caffe model to the `models` folder:
+1. :ref:`Build Samples <build-samples>`.
+2. :ref:`Select a Sample <select-sample>`.
+3. :ref:`Download a suitable model <download-model>`.
+4. :ref:`Download media files used as input, if necessary <download-media>`.
 
-   .. tab-set::
+Once you perform all the steps, you can :ref:`run inference with the chosen sample application <run-inference>` to see the results. 
 
-      .. tab-item:: Windows
-         :sync: windows
+.. _build-samples:
 
-         .. code-block:: bat
+Build the Sample Applications
+================================
 
-            omz_downloader --name googlenet-v1 --output_dir %USERPROFILE%\Documents\models
+Select a sample you want to use from the :doc:`OpenVINO Samples <openvino_docs_OV_UG_Samples_Overview>` page, and follow the instructions below to build it in your operating system.
 
-      .. tab-item:: Linux
-         :sync: linux
-
-         .. code-block:: sh
-
-            omz_downloader --name googlenet-v1 --output_dir ~/models
-
-      .. tab-item:: macOS
-         :sync: macos
+.. note::
 
-         .. code-block:: sh
+   Some samples may also require `OpenCV <https://github.com/opencv/opencv/wiki/BuildOpenCV4OpenVINO>`__ to run properly. Make sure to install it for use with vision-oriented samples. 
 
-            omz_downloader --name googlenet-v1 --output_dir ~/models
+Instructions below show how to build sample applications with CMake. If you are interested in building them from source, check the `build instructions on GitHub <https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/build.md>`__ .
 
+.. tab-set::
 
-   Your screen will look similar to this after the download and show the paths of downloaded files:
+   .. tab-item:: Linux
+      :sync: linux
 
-   .. tab-set::
+      The officially supported Linux build environment is the following:
```

Comment:
this is a duplication of common system requirements. Should we just refer to a generic page with system requirements?
As far as I remember, @kblaszczak-intel was working on such page

Reply:
If you think the official OV docs system requirements page is a proper substitute for this section, we would gladly remove it. The less superficial content the better.